VF Page/Exercise 3/RadioControllor.page-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexPage xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>48.0</apiVersion>
+    <availableInTouch>false</availableInTouch>
+    <confirmationTokenRequired>false</confirmationTokenRequired>
+    <label>RadioControllor</label>
```

Comment:
VF page name should be meaningful to the work it is doing. some thing like 
SelectBillToContact or UpdateBillToContact